mmv1/products/gkebackup/RestorePlan.yaml

```diff
@@ -496,6 +547,87 @@ properties:
                     description: |
                       A string that specifies the desired value in string format
                       to use for transformation.
+      - !ruby/object:Api::Type::Array
```

Comment:
Can you also add update test for these new fields? Thanks.

Reply:
Added. PTAL.